WordPressAuthenticator/Unified Auth/View Related/Sign up/SignupMagicLinkViewController.swift

```diff
@@ -0,0 +1,197 @@
+import UIKit
+
+
+/// SignupMagicLinkViewController: step two in the signup flow.
+/// This VC prompts the user to open their email app to look for the magic link we sent.
+///
+final class SignupMagicLinkViewController: LoginViewController {
+
+    // MARK: Properties
+
+    @IBOutlet private weak var tableView: UITableView!
+    private var rows = [Row]()
+    private var errorMessage: String?
+    private var shouldChangeVoiceOverFocus: Bool = false
+
+    var emailMagicLinkSource: EmailMagicLinkSource?
+    override var sourceTag: WordPressSupportSourceTag {
+        get {
+            if let emailMagicLinkSource = emailMagicLinkSource,
+                emailMagicLinkSource == .signup {
+                return .wpComSignupMagicLink
+            }
+            return .loginMagicLink
```

Comment:
There's going to be a separate Login Magic Link VC, right? So I think this sourceTag will always be signup. At least, it should never return login.